docs/platforms/javascript/common/tracing/index.mdx

```diff
@@ -7,6 +7,8 @@ sidebar_order: 4000
 With [tracing](/product/performance/), Sentry tracks your software performance, measuring metrics like throughput and latency, and displaying the impact of errors across multiple systems. Sentry captures distributed traces consisting of transactions and spans, which measure individual services and individual operations within those services. Learn more about our model in [Distributed Tracing](/product/sentry-basics/tracing/distributed-tracing/).
 
 <PlatformCategorySection supported={["server", "serverless"]}>
+  You can additionally <PlatformLink to="/profiling">set up Profiling</PlatformLink> to get even more detailed tracing information like stack traces and flame graphs.
```

Comment:
I think we have this page on browser too, can we move this out of the server/serverless check?